src/providers/LayerProvider/LayerProvider.tsx

```diff
@@ -360,6 +366,7 @@ export const LayerProvider = ({
           read,
           syncTimestamps,
           readTimestamps,
+          expireDataCaches: resetCaches,
```

Comment:
Renamed this `expiration` to more tightly correlate with what this does - leaves the caches in place just marks them out of date.